src/plugins/data_views/common/constants.ts

```diff
@@ -22,15 +22,16 @@ export const RUNTIME_FIELD_TYPES = [
 ] as const;
 
 /**
- * Used to determine if the instance has some user created index patterns by filtering index patterns
- * that are created and backed only by Fleet server data
- * Should be revised after https://github.com/elastic/kibana/issues/82851 is fixed
- * For more background see: https://github.com/elastic/kibana/issues/107020
+ * Used to optimize on-boarding experience to determine if the instance has some user created data views or data indices/streams by filtering data sources
+ * that are created by default by elastic in ese.
+ * We should somehow prevent creating initial data for the users without their explicit action
+ * instead of relying on these hardcoded assets
```

Comment:
100% agree, but until we do that, can we return the old data view names?

Reply:
Do you mean to keep the older logic from `hasUserDataView`? As I understand, we don't need that because https://github.com/elastic/kibana/issues/82851 was fixed, and we should have to remove it in https://github.com/elastic/kibana/issues/127869. 

Do you think there is a reason to keep it? If yes, then I'd also revert https://github.com/elastic/kibana/issues/127869

Reply:
Just the old data view names, `logs-*`, `logs-elastic_agent` etc